scripts/scil_compute_ihMT_maps.py

```diff
@@ -0,0 +1,442 @@
+
+Maps :
+    Magnetization Transfer (MTR) and inhomogeneous (ihMTR) Ratio maps
+          The (ih)MT ratio is a measure reflecting the amount of bound protons.
+    Magnetization Transfer (MTsat) and inhomogeneous (ihMTsat) saturation maps
+          The (ih)MT saturation is a pseudo-quantitative ihMT maps representing
+          the signal change between the bound and free water pools.
+
+"""
+
+import argparse
+import os
+
+import json
+import nibabel as nib
+import numpy as np
+import scipy.ndimage
+
+from scilpy.io.image import get_data_as_mask
+from scilpy.io.utils import (add_overwrite_arg, assert_inputs_exist,
+                             assert_output_dirs_exist_and_empty)
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('id_subj',
+                   help='Name of Subject for saving output files.')
+    p.add_argument('out_dir',
+                   help='Path to output folder.')
+    p.add_argument('in_mask',
+                   help='Path to the T1 binary brain mask. Must be the sum '
+                        'of the three tissue probability maps from '
+                        'T1 segmentation (GM+WM+CSF).')
+    p.add_argument('--filtering', action='store_true',
+                   help='Gaussian filtering to remove Gibbs ringing.'
+                        'Not recommanded.')
+
+    g = p.add_argument_group(title='ihMT contrasts', description='Path to '
+                             'echoes corresponding to contrasts images. All '
+                             'constrat must have the same number of echoes '
+                             'and coregistered between them.'
+                             'Use * to include all echoes.')
+    g.add_argument('--in_altnp', nargs='+',
+                   help='Path to all echoes corresponding to the '
+                        'alternation of Negative and Positive'
+                        'frequency saturation pulse.')
+    g.add_argument('--in_altpn', nargs='+',
+                   help='Path to all echoes corresponding to the '
+                        'alternation of Positive and Negative '
+                        'frequency saturation pulse.')
+    g.add_argument("--in_mtoff", nargs='+',
+                   help='Path to all echoes corresponding to the '
+                        'no frequency saturation pulse (reference image).')
+    g.add_argument("--in_negative", nargs='+',
+                   help='Path to all echoes corresponding to the '
+                        'Negative frequency saturation pulse.')
+    g.add_argument("--in_positive", nargs='+',
+                   help='Path to all echoes corresponding to the '
+                        'Positive frequency saturation pulse.')
+    g.add_argument("--in_t1w", nargs='+',
+                   help='Path to all echoes corresponding to the '
+                        'T1-weigthed.')
```

Comment:
Are all these files present when we convert our MyeloInferno protocol? If so, maybe mention it somewhere that the bids conversion or dcm2niix conversion will create all these files. 

And I think we will need a little documentation in docs/ to come with this script. It's so novel and far from dMRI that we are used to do.